catalog/app/components/StackedAreaChart/StackedAreaChart.js

```diff
@@ -133,22 +141,36 @@ export default function MultiSparkline({
         {!!onCursor && cursorPaint.def}
       </defs>
       <g>
-        {areas.map(
-          (points, i) =>
-            points && (
-              <polygon
-                // eslint-disable-next-line react/no-array-index-key
-                key={`area:${i}`}
-                onMouseEnter={handleAreaEnter(i)}
-                onMouseLeave={handleAreaLeave}
-                points={SVG.pointsToSVG(points)}
-                fill={areaPaints[i].ref}
-                fillOpacity={
-                  onCursor && cursor && cursor.i != null && cursor.i !== i ? fade : 1
-                }
-              />
-            ),
-        )}
+        {figures.map(({ area, stroke }, i) => (
+          <g
+            // eslint-disable-next-line react/no-array-index-key
+            key={`area:${i}`}
+            onMouseEnter={handleAreaEnter(i)}
+            onMouseLeave={handleAreaLeave}
+          >
+            <polygon
+              points={SVG.pointsToSVG(area)}
+              fill={
+                // If color is #COLOR7
+                areaPaints[i].ref.length === 7
```

Comment:
i dont get it. what's special about color 7?

Reply:
```
#COLORX
1234567
```

Reply:
#ABCDEF - hex color is a string 7 characters long

Reply:
Another variant is `url(#key)`, and it throws error when passed to `lighten()` function

Reply:
oh i see